src/Sarif/Sarif.csproj

```diff
@@ -99,7 +100,13 @@
       <DependentUpon>SdkResources.resx</DependentUpon>
     </Compile>
   </ItemGroup>
-
-  <Import Project="ToDotNet\ToDotNet.targets" />
+  
+<!-- Disabling the automatic code generation. This code emit is generating
```

Comment:
>Disabling [](http://example.com/codeflow?start=5&length=9)

Everyone: please note this change. The SARIF OM has been stable for years now. We don't need to keep updating it. We can't enforce stringent style guidelines on that code if the autogen keeps overwriting changes again (and I don't feel like investing in fixing the JSchema codegen for these style issues, though that would be nice if someone did it).